packages/ethash/src/util.ts

```diff
@@ -40,8 +40,8 @@ export function getFullSize(epoc: number) {
   return sz
 }
 
-export function getEpoc(blockNumber: number) {
-  return Math.floor(blockNumber / exports.params.EPOCH_LENGTH)
+export function getEpoc(blockNumber: bigint) {
+  return Math.floor(Number(blockNumber) / exports.params.EPOCH_LENGTH)
```

Comment:
if we would rather protect blockNumber as bigint here, I believe we can do this, since bigint division will implicitly truncate decimal digits 
```suggestion
  return blockNumber / BigInt(exports.params.EPOCH_LENGTH)
```